src/components/SearchBox.tsx

```diff
@@ -109,6 +111,10 @@ export default class SearchBox<OptionType> extends React.PureComponent<
     );
   };
 
+  componentDidMount() {
+    setTimeout(() => this.inputRef.current?.focus(), 100);
```

Comment:
why timeout 100?

Reply:
a small timeout was needed , maybe something was taking too long to render
didn't work without a timeout